share/functions/export.fish

```diff
@@ -9,7 +9,7 @@ function export --description 'Set global variable. Alias for set -gx, made for
             case 1
                 set -gx $v $$v
             case 2
-                if test $v[1] = PATH
+                if contains $v[1] PATH CDPATH MANPATH
```

Comment:
If $v[1] starts with a "-", it'll be interpreted as an option. contains will fail if it doesn't know the option and succeed if it does, and additionally will print a message.

`contains -- $v[1]` is what you want.